### PR TITLE
Needs more analysis section in Summary

### PR DESCRIPTION
**Pull request by wittejm**
A few unfinished bits: the in-page link by ambiguous_charge_id is using a gross hack at the moment . And I haven't removed the old Questions banner. Just pushing what I have for now though.

### CODE REVIEW

File: src/frontend/src/components/RecordSearch/Record/RecordSummary/ChargesList.tsx [hunk 1]
```diff
@@ -38,7 +41,7 @@ export default class ChargesList extends React.Component<Props> {
   buildListItems(chargesNames: string[]) {
     const listItems = chargesNames.map(((chargeName: string, index:number) => {
         return (
-            <li key={"chargeItem" + index} className="bt b--light-gray pt1 mb1">{chargeName}</li>
+            <li key={"chargeItem" + index} className="bt b--light-gray pt1 mb1"><a href={"#" + chargeName[0]}>{chargeName[1]}</a></li>
```
Comment (hmarcks) [inline src/frontend/src/components/RecordSearch/Record/RecordSummary/ChargesList.tsx:44]:
```suggestion
            <li key={"chargeItem" + index} className="bt b--light-gray pt1 mb2"><a href={"#" + chargeName[0]} className="link hover-blue>{chargeName[1]}</a></li>
```

**Comment by KentShikama** [comment]
@michaelzhang43 @hmarcks @wittejm So just to double check, this PR will remove the questions banner (bottom half of screenshot) and rely on the absence of cases in the NMA (top half of screenshot) to tell the user all disambiguating questions have been answered.

Screen Shot 2020-05-16 at 6 28 55 PM

**Comment by michaelzhang43** [comment]
@KentShikama that sounds good. No need to show redundant info

**Comment by KentShikama** [comment]
Resolves https://github.com/codeforpdx/recordexpungPDX/issues/1105

**Comment by wittejm** [comment]
@michaelzhang43 If there are no charges that need more analysis, should we hide the section rather than show the section with "None"?

showing_nma_section

**Comment by michaelzhang43** [comment]
Yes, good point

**Pull request merged**